build.gradle.kts

```diff
@@ -72,6 +72,8 @@ val apiSpecTitle = "NIAEFEUP Website - Backend API specification"
 val apiSpecDescription =
     """This specification documents the available endpoints and possible operations on the website's backend.
         |For each of the operations, its purpose, security, requests and possible responses are documented.
+        |
+        |Postman collection also available <a href="postman-collection.json" download>here</a>.
```

Comment:
Where does this link?

Reply:
The openAPI specification and the postman collection files are copied to the same directory by the `generateDocs` task. Netlify is serving the docs folder created by the `generateDocs` task, which, after the redoc UI is built with the respective command, has 3 files: the HTML redoc page, the openapi3.json file and the postman-collection.json. This anchor tag is parsed in the redoc HTML file, so it links to the postman collection in the same folder. You can check it out in the preview and in the repo ReadMe. It also works locally.
https://github.com/NIAEFEUP/website-niaefeup-backend/blob/2278ead10e4c1eeb25e332c64e44ed7469f6a766/README.md?plain=1#L81